server/app/category.go

```diff
@@ -34,10 +36,8 @@ func (a *App) CreateCategory(category *model.Category) (*model.Category, error)
 }
 
 func (a *App) UpdateCategory(category *model.Category) (*model.Category, error) {
-	// set to default category, UI doesn't create with Type
-	if strings.TrimSpace(category.Type) == "" {
```

Comment:
Note to reviewer: this is now being done in `category.Hydrate()` call.

Reply:
In the `Category.Hydrate` is not doing the `string.TrimSpace`, I guess that is ok, but just to let you know that it is not exactly equivalent.